core/base/mergeTreeDistanceMatrix/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+ttk_add_base_library(mergeTreeDistanceMatrix
+  SOURCES
+    MergeTreeDistanceMatrix.cpp
+  HEADERS
+    MergeTreeDistanceMatrix.h
+  DEPENDS
+    triangulation
```

Comment:
I don't believe that dependence is needed.

Reply:
Thanks! It will be removed in the next commit.